app/renderer/components/bookmarks/bookmarksToolbar.js

```diff
@@ -196,23 +195,23 @@ class BookmarksToolbar extends React.Component {
   mergeProps (state, ownProps) {
     const currentWindow = state.get('currentWindow')
     const activeFrame = frameStateUtil.getActiveFrame(currentWindow) || Immutable.Map()
+    const bookmarkDisplayMode = bookmarkUtil.showOnlyFavicon()
+        ? 1
+        : bookmarkUtil.showFavicon()
+          ? 2
+          : 3
 
     const props = {}
     // used in renderer
     props.shouldAllowWindowDrag = !isWindows && windowState.shouldAllowWindowDrag(state, currentWindow, activeFrame, isFocused(state))
     props.toolbarBookmarks = bookmarksState.getBookmarksWithFolders(state, 0).take(110).map(item => item.get('key'))
-
+    props.textOnly = (bookmarkDisplayMode === 3)
```

Comment:
if we follow the above comment here we could have

```
props.textOnly = (bookmarkDisplayMode === bookmarkDisplayOptions.textOnly)
```

or any other more meaningful name

Reply:
The purpose of this prop is already to make a readable name: `textOnly` as that value *is* used in other places in the component, whereas the exact value of `bookmarkDisplayMode` is not considered anywhere else.